draft-iab-bias-workshop-report.md

```diff
@@ -328,10 +328,10 @@ Some key takeaways from the workshop are -
 - There is a need for the technical community to address the management gaps in operating Community Networks.
 - Work should be done in documenting best practices for operating Community Networks.
 - During the development of protocols, explicit manageability considerations related to Community Networks should be considered.
-- IETF can play a key role in building consensus on solutions that foster digital inclusion.
-- IETF should continue to work on protocol enhancements that protect user privacy.
-- IETF should continue to work on protocols or protocl extensions that enable more transparency on filtering and promote their use and deployment.
-- IETF should continue to work on VPN-like services and potentially support measurements to understand their deployment and use.
+- There is a need to build consensus on solutions that foster digital inclusion.
```

Comment:
Actually, looking on this again, I'm not sure I fully understand this point. Why is this about build consensus (rather than building solutions that support digital inclusion)?

Reply:
It comes from the comment made that IETF is a good venue for consensus building to ensure experts are on the same page and then promote that policy and government! Earlier I did not want to mention "promote" because we started with IETF should but now this can be reworded. How about -- 
- Build consensus on solutions that have the most significant impact in fostering digital inclusion. Further, promoting these solutions ensures that efforts to bridge the digital divide are effective and inclusive.

Reply:
yes better